src/main/java/com/iemr/tm/service/videoconsultation/VideoConsultationServiceImpl.java

```diff
@@ -19,103 +19,99 @@
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see https://www.gnu.org/licenses/.
 */
-package com.iemr.tm.service.swymed;
+package com.iemr.tm.service.videoconsultation;
 
 import java.util.ArrayList;
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import com.iemr.tm.data.swymed.UserJitsi;
-import com.iemr.tm.data.swymed.UserSwymed;
+import com.iemr.tm.data.videoconsultation.UserJitsi;
+import com.iemr.tm.data.videoconsultation.UserVideoConsultation;
 import com.iemr.tm.repo.login.MasterVanRepo;
-import com.iemr.tm.repo.swymed.UserJitsiRepo;
-import com.iemr.tm.repo.swymed.UserRepo;
-import com.iemr.tm.repo.swymed.UserSwymedRepo;
+import com.iemr.tm.repo.videoconsultation.UserJitsiRepo;
+import com.iemr.tm.repo.videoconsultation.UserRepo;
+import com.iemr.tm.repo.videoconsultation.VideoConsultationUserRepo;
 import com.iemr.tm.utils.config.ConfigProperties;
-import com.iemr.tm.utils.exception.SwymedException;
+import com.iemr.tm.utils.exception.VideoConsultationException;
 
 @Service
-public class SwymedServiceImpl implements SwymedService {
+public class VideoConsultationServiceImpl implements VideoConsultationService {
 
-	// swymed://dnsname?l=mylogin&p=mypassword&d=mydomain&c=callnumber&m=1
-	private String swymed_dnsname = ConfigProperties.getPropertyByName("swymed_dnsname");
+	private String videoConsultationDNSName = ConfigProperties.getPropertyByName("videoConsultationDNSName");
```

Comment:
ConfigProperties.getPropertyByName("swymed_dnsname");
swymed_dnsname is a property mentioned in application.properties.